### PR TITLE
feat(cb2-7009): Light vehicles test group 1

### PR DESCRIPTION
**Pull request by leifkemp-bjss**
## CB2-7009: Create Light Vehicles Test Group 1

_PR to add contingency test group 1 for Light Vehicles to VTM, also adds the 3 new vehicle types to the tech records enum. Switch cases regarding checking vehicle type for tech record components have been given a default case until the tech team start work on the tech records._
[CB2-7009](https://dvsa.atlassian.net/browse/CB2-7009)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Necessary `id` required prepended with `"test-"` have been checked with automation testers and added
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Link to the PR added to the repo
- [ ] Delete branch after merge

### CODE REVIEW

File: src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts [hunk 1]
```diff
@@ -120,6 +120,9 @@ export class TechRecordSummaryComponent implements OnInit {
         return this.getHgvTemplates();
       case VehicleTypes.TRL:
         return this.getTrlTemplates();
+      // TODO: Double check this
+      default:
+        return this.getPsvTemplates();
```
Comment (leifkemp-bjss) [inline src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts:125]: There are no light vehicle template functions in the tech records at the moment, so I've just had them default to the PSV functions

Comment (naathanbrown) [reply src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts:125]: Proper pedantic but can you make the TODO slightly clearer

Comment (naathanbrown) [reply src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts:125]: Oh this is a draft, so it's fine

Comment (leifkemp-bjss) [reply src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts:125]: They've been rewritten now

File: src/app/forms/templates/test-records/section-templates/required/contingency-required-hidden-section-lgv-car.template.ts [hunk 2]
```diff
@@ -0,0 +1,229 @@
+import { FormNode, FormNodeEditTypes, FormNodeTypes, FormNodeViewTypes } from '@forms/services/dynamic-form.types';
+
+export const CreateRequiredSectionLGVCar: FormNode = {
```
Comment (tahayazici) [inline src/app/forms/templates/test-records/section-templates/required/contingency-required-hidden-section-lgv-car.template.ts:3]:
```suggestion
export const CreateRequiredSectionLgvCar: FormNode = {
```

File: src/app/forms/templates/test-records/section-templates/required/contingency-required-hidden-section-lgv-car.template.ts [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+    {
+      name: 'testStatus',
+      type: FormNodeTypes.CONTROL,
+      label: 'Test status',
+      viewType: FormNodeViewTypes.HIDDEN,
+      editType: FormNodeEditTypes.HIDDEN,
+      options: [
+        { label: 'Submitted', value: 'submitted' },
+        { label: 'Cancelled', value: 'cancelled' }
+      ]
```
Comment (gjulien-bjss) [inline src/app/forms/templates/test-records/section-templates/required/contingency-required-hidden-section-lgv-car.template.ts:28]: don't need options on a hidden field surely?

File: src/app/forms/templates/test-records/section-templates/required/contingency-required-hidden-section-lgv-car.template.ts [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+    {
+      name: 'shouldEmailCertificate',
+      type: FormNodeTypes.CONTROL,
+      label: 'Should email certificate?',
+      viewType: FormNodeViewTypes.HIDDEN,
+      editType: FormNodeEditTypes.HIDDEN,
+      options: [
+        { label: 'Yes', value: 'yes' },
+        { label: 'No', value: 'no' }
+      ]
```
Comment (gjulien-bjss) [inline src/app/forms/templates/test-records/section-templates/required/contingency-required-hidden-section-lgv-car.template.ts:148]: same

File: src/app/forms/templates/test-records/section-templates/required/contingency-required-hidden-section-motorcycle.template.ts [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+    {
+      name: 'testStatus',
+      type: FormNodeTypes.CONTROL,
+      label: 'Test status',
+      viewType: FormNodeViewTypes.HIDDEN,
+      editType: FormNodeEditTypes.HIDDEN,
+      options: [
+        { label: 'Submitted', value: 'submitted' },
+        { label: 'Cancelled', value: 'cancelled' }
+      ]
```
Comment (gjulien-bjss) [inline src/app/forms/templates/test-records/section-templates/required/contingency-required-hidden-section-motorcycle.template.ts:28]: same

File: src/app/forms/templates/test-records/section-templates/required/contingency-required-hidden-section-motorcycle.template.ts [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+    {
+      name: 'shouldEmailCertificate',
+      type: FormNodeTypes.CONTROL,
+      label: 'Should email certificate?',
+      viewType: FormNodeViewTypes.HIDDEN,
+      editType: FormNodeEditTypes.HIDDEN,
+      options: [
+        { label: 'Yes', value: 'yes' },
+        { label: 'No', value: 'no' }
+      ]
```
Comment (gjulien-bjss) [inline src/app/forms/templates/test-records/section-templates/required/contingency-required-hidden-section-motorcycle.template.ts:136]: same

**Review by gjulien-bjss — APPROVED** [review]

**Review by tahayazici — APPROVED** [review]

**Pull request merged**